IfScooter/IfScooters/ScooterRent.cs

```diff
@@ -0,0 +1,77 @@
+﻿using System;
+
+namespace IfScooters
+{
+    public class ScooterRent : IRentalCompany
+    {
+        public static void Main() { } //Bez šī man neļāva palaist neko
```

Comment:
jāmaina tips uz class library projektam no console app.. :) tad ies. 
šis jāizņem ārā.